### DISCUSSION TITLE
URGENT ! Downloading prisma engines Error

### DISCUSSION THREAD

**Discussion by MuhammadAli13562** [prisma/prisma#23959] [Q&A] [answered]
### Question

I have tried other solutions by configuring ipv4 to automatic DHCP but it did not work. It was working fine the other day but now anything with a prisma command gives the error

command : `npx prisma genereate`
```
Downloading Prisma engines for Node-API for debian-openssl-1.1.x [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/b9a39a7ee606c28e3455d0fd60e78c3ba82b1a2b/debian-openssl-1.1.x/libquery_engine.so.node.gz.sha256 failed, reason:
```

Urgent help needed. I am completely stuck.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @MuhammadAli13562 👋 

This error typically occurs due to network issues that prevent Prisma from downloading the necessary binaries. Can you try using a different internet connection, mobile hotspot, or VPN to see if this resolves the issue. 

Can you try to open and download the URL (https://binaries.prisma.sh/all_commits/b9a39a7ee606c28e3455d0fd60e78c3ba82b1a2b/debian-openssl-1.1.x/libquery_engine.so.node.gz.sha256) manually in a browser on your machine to see if it downloads successfully?

**Reply by MuhammadAli13562** [reply]
Yes its perfectly downloading file from this link everytime i try.

**Comment by Jolg42** [comment]
@MuhammadAli13562 I have a few questions:
- Are you using a VPN or any special network configuration?
- Did it work before?
- Which version of Prisma CLI are you using?
- Could you run the following command `npx prisma debug` and share the output here?

**Reply by MuhammadAli13562** [reply, marked as answer]
Yes it was working just fine yesterday but today made some changes to schema , tried running generate and the error was thrown.

Now Solved !
Solution : Weirdly enough , I tried using a VPN and i connected it once and then disconnected it and after that everything is working fine. I dont know what was the problem but now everything is working just after using vpn once.
